ops/cli.py

```diff
@@ -86,35 +82,55 @@ def __call__(self) -> int:
         return getattr(self, args.command)(sys.argv[2:])
 
     def _usage(self) -> str:
-        msg, space = "\n", 20
+        max = 0
+        for command in sorted(self.registered_commands):
+            if len(command) > max:
+                max = len(command)
+        msg = "\n"
         for command in sorted(self.registered_commands):
-            msg += f"   {command}{'  ' * (space - len(command))} |-> {getattr(self, command).__doc__}\n"
+            msg += f"  {command}{' ' * (max - len(command))}"
+            msg += f" |-> {getattr(self, command).__doc__}\n"
         return msg
 
+    @staticmethod
+    def _subprocess_call(command: str):
+        """Run COMMAND in a subprocess."""
+        print(f"Running: {command}")
+        return subprocess.check_call(command, shell=True)
+
+    @staticmethod
+    def _subprocess_call_unchecked_because_who_cares_if_it_fails(
+            command: str
+    ):
```

Comment:
Maybe dial down the sass...
```suggestion
    def _subprocess_call_unchecked( command: str ):
```